### PR TITLE
[Ansible Installer] Add Kiali as addon

### PR DESCRIPTION
**Pull request by gbaufake**
Including Kiali on Ansible Installer in order to match #5683

### CODE REVIEW

File: install/ansible/istio/tasks/install_addons.yml [hunk 1]
```diff
@@ -20,4 +20,22 @@
...
+    curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | IMAGE_NAME=kiali/kiali IMAGE_VERSION=latest NAMESPACE=istio-system VERSION_LABEL=master VERBOSE_MODE=4 envsubst | {{ cmd_path }} create -n istio-system -f -
+  when:
+    - is_istioaddon_iterable
+    - "'kiali' in istio.addon"
+    - "cluster_flavour == 'ocp'"
+
+- name: Install Kiali on Kubernetes
+  shell: |
+   curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/kubernetes/kiali-configmap.yaml | VERSION_LABEL=master envsubst | {{ cmd_path }} create -n istio-system -f -
+   curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/kubernetes/kiali.yaml | IMAGE_NAME=kiali/kiali IMAGE_VERSION=latest NAMESPACE=istio-system VERSION_LABEL=master  VERBOSE_MODE=4 envsubst | {{ cmd_path }} create -n istio-system -f -
```
Comment (gyliu513) [inline install/ansible/istio/tasks/install_addons.yml]: the `latest` may not work sometime, can we use a released version here?

Comment (gbaufake) [reply install/ansible/istio/tasks/install_addons.yml]: we can use released version.

do you have something version in mind?

Comment (gbaufake) [reply install/ansible/istio/tasks/install_addons.yml]: @gyliu513  set version to 0.3.0.Alpha.

**Comment by gbaufake** [comment]
@gyliu513 can you review the new changes?

**Comment by gyliu513** [comment]
/cc @sdake @linsun @costinm 

Can you help check if we can merge? Thanks.

**Comment by gbaufake** [comment]
@sdake @linsun  or @costinm 
Can you review it?

**Pull request merged**